help-docs: Fix instructions for the "Logging out" mobile app feature.

drrosa opened the pull request:
The logging-out instructions haven't been accurate for a long while.

- Fixes the instructions and replaces the iOS and Android tabs with a single Mobile tab plus a new macro with instructions for accessing the profile menu.

```
$ git grep "ios"
logging-out.md:{tab|ios}

$ git grep "android"
logging-out.md:{tab|android}
```

- Notice that there aren't any references left to `ios` and `android`, but they shouldn't be removed from `tabbed_sections.py` as we may need them to document [#mobile > platform differences](https://chat.zulip.org/#narrow/stream/48-mobile/topic/platform.20differences).

**Screenshots and screen captures:**
<img width="527" alt="image" src="https://user-images.githubusercontent.com/2343554/178015235-dfdfcf39-6ff6-4c1d-b1d1-1054c4aea5ea.png">

- Replaces step one with a macro for reusability.
```
$ git grep "tap your \*\*profile picture\*\* " 

logging-in.md:1. From the home screen, tap your **profile picture** in the lower right.
switching-between-organizations.md:1. From the home screen, tap your **profile picture** in the lower right.
```
- Reorders tabs Web > Desktop > Mobile for consistency.

<img width="575" alt="image" src="https://user-images.githubusercontent.com/2343554/178015372-1381a0fe-9087-4468-97ae-a91a4e2ea1c6.png">

<img width="540" alt="image" src="https://user-images.githubusercontent.com/2343554/178015433-27ceb5e6-839c-484b-bcd2-41cfc44b109e.png">


<details>
<summary>Self-review checklist</summary>

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability

Communicate decisions, questions, and potential concerns.

- [x] Explains differences from previous plans.
- [x] Highlights technical choices and bugs encountered.
- [x] Calls out remaining decisions and concerns.
- [x] Automated tests

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [x] Strings.
- [x] End-to-end functionality of buttons, interactions, and flows.
- [x] Corner cases, error conditions, and easily imagined bugs.
</details>

drrosa commented:
@alya ready for review.

alya commented on templates/zerver/help/include/mobile-profile-menu.md:
```diff
@@ -0,0 +1 @@
+1. From the home screen, tap your **profile picture** in the lower right.
```
Hm, I don't think the "home screen" is a well-defined concept... We have the bottom nav on a number of screens, some of which I definitely would not call a home screen (e.g. streams)

It might be best to just say, "Tap your **profile picture** in the lower right," or maybe "Tap your **profile picture** in the bottom right corner."

@gnprice what do you think?

drrosa replied:
Good point. Yeah, "Tap your profile picture in the bottom right corner of the screen." would be best, I think.

alya replied:
I know we had "corner of the screen" phrasing elsewhere, but I wonder if we should standardize on "corner of the app"? E.g. the user might be doing a split screen...

drrosa replied:
Yes, I think it'd be a good idea to standardize on "corner of the app" because it's more specific and we can use it to refer to the web app, desktop app, or mobile app.

alya replied:
Cool, could you check for and fix any uses of `screen` if you haven't done so already?

alya commented on templates/zerver/help/logging-out.md:
```diff
@@ -6,21 +6,13 @@
-
-2. Tap **Log out**.
+1. Tap **Log out**.
 
```
Let's add an instruction for the confirmation dialog. Something like:

> 1. Tap **Log out** to confirm.

This one will have to be updated following up on the changes discussed in https://chat.zulip.org/#narrow/stream/48-mobile/topic/confirm.2Fcancel.20modals

alya replied:
Sounds good to me!

alya commented on templates/zerver/help/logging-in.md:
```diff
@@ -33,7 +38,7 @@ and GitHub account use the same email address.
 
```
Let's improve the above instructions while we're here:

> 1. Open the **left sidebar** (`Ctrl` + `Shift` + `s`).

Let's direct people to the `+` if the sidebar is open, or the Zulip > Add organization menu as something that always works. That seems better than teaching an obscure keyboard shortcut.

> 1. Click the **plus** (+) icon.

This should use the `+` icon (https://zulip.readthedocs.io/en/latest/documentation/helpcenter.html#icons).

alya replied:
> 1. Set your [proxy settings](/help/connect-through-a-proxy) or add a
   [custom certificate](/help/custom-certificates) if needed (rare).

Since this is rare, let's put it in a warning block instead of the main instructions flow. Do those pages have any explanation for when you might need to do this?

drrosa replied:
> Let's improve the above instructions while we're here:

<img width="532" alt="image" src="https://user-images.githubusercontent.com/2343554/178773315-d94c8975-8a80-4b0d-a3ff-370ce254e26b.png">

What do you think?

The tip could be a macro that we can put in other docs that reference the left sidebar in the Desktop app.

alya replied:
> Set your [proxy settings](https://github.com/help/connect-through-a-proxy) or add a
[custom certificate](https://github.com/help/custom-certificates) if needed (rare).

The placement looks good, but let's try rewording to something like:

> If you are having trouble connecting, you may need to set your [proxy settings](https://github.com/help/connect-through-a-proxy) or add a [custom certificate](https://github.com/help/custom-certificates).

alya replied:
I like the overall approach, but I have some feedback on the other parts of the screenshot as well... Could you push the code so that it's easier to fiddle with? 

In general, whenever you post a screenshot of a proposed change, it helps to include the code (either pushed to the PR or in the comment), so that I don't have to retype text from an image in order to propose edits. :)

drrosa replied:
Oh makes sense, sorry about that! I'll push the code :)

alya commented on templates/zerver/help/switching-between-organizations.md:
```diff
@@ -4,6 +4,12 @@ This article assumes you've [logged in](/help/logging-in) to each organization a
+
 {tab|desktop}
 
 1. Open the **left sidebar** (`Ctrl` + `Shift` + `s`).
```
Let's clarify that you only have to do this if the left sidebar isn't open to begin with.

drrosa commented:
@alya thanks for the review! I've made the requested changes plus a few other tweaks and pushed the new code.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/2343554/179137519-7ca1684f-1e07-44ec-af57-8e0380a351a9.png">

<img width="528" alt="image" src="https://user-images.githubusercontent.com/2343554/179142010-805f500f-332d-4433-918c-0d458da4a4a5.png">

<img width="532" alt="image" src="https://user-images.githubusercontent.com/2343554/179138539-3069b342-fd74-4c22-b3a2-717d867181ff.png">

<img width="532" alt="image" src="https://user-images.githubusercontent.com/2343554/179138296-b25d1f6e-dac1-4117-8a0d-ff42fbfdfb3b.png">

<img width="572" alt="image" src="https://user-images.githubusercontent.com/2343554/179137895-f1014d1e-dc2b-4fb7-9a79-e5737ca1e4e7.png">

<img width="608" alt="image" src="https://user-images.githubusercontent.com/2343554/179137428-cd35f147-1ff0-4cc2-b7e2-8ee6487743de.png">

alya commented on templates/zerver/help/include/desktop-toggle-sidebar-tip.md:
```diff
@@ -0,0 +1,4 @@
+!!! tip ""
+
+    In order to show or hide the **leftmost sidebar**, you can click on
+    **View** > **Toggle Sidebar** in the main menu bar or press `Ctrl` + `Shift` + `s`.
```
How about something like this?

> To show or hide the **organizations sidebar** on the left, select **Toggle Sidebar** from the **View** menu in the top menu bar.

The keyboard shortcut is shown in the menu, so I don't think we need to explicitly teach it here.

alya commented on templates/zerver/help/logging-in.md:
```diff
@@ -13,27 +13,34 @@ how you signed up. E.g. if you originally signed up using your Google
+1. Click the **plus** (<i class="fa fa-plus"></i>) icon on the far left side of the app. You can also click on **Zulip** > **Add organization** in the main menu bar.
 
-1. Enter your Zulip URL, and click **Connect**.
+1. Enter your Zulip URL and click **Connect**.
```
Hm, what's the general pattern regarding  Oxford commas in the help center? I'm not sure we want to drop it.

alya commented:
Since we're touching these articles, could you please go through and make sure these articles are all listed in each other's **Related articles** sections (in our standard format)?

[Logging in](https://zulip.com/help/logging-in)
[Logging out](https://zulip.com/help/logging-out)
[Switching between organizations](https://zulip.com/help/switching-between-organizations)
[Deactivate your account](https://zulip.com/help/deactivate-your-account)

Thanks!

alya commented on templates/zerver/help/logging-in.md:
```diff
@@ -13,27 +13,34 @@ how you signed up. E.g. if you originally signed up using your Google
 and GitHub account use the same email address.
 
-
 ### Log in to a Zulip organization for the first time
```
Please update the headers in this article so that the top ones are `##` level, and the next ones are `###`. We have been cleaning up all the places where `##` headers are skipped.

alya commented on templates/zerver/help/switching-between-organizations.md:
```diff
@@ -4,30 +4,28 @@ This article assumes you've [logged in](/help/logging-in) to each organization a
+{tab|desktop}
 
-!!! warn ""
+1. Click on a profile picture from the list of Zulip organizations on the far left side of the app.
```
Let's make this:

> Click on a logo in the **organizations sidebar** on the left.

drrosa commented:
I changed the wording based on your feedback, and added a new commit to fix all occurrences of "screen", including inconsistent use of "lower/upper" vs. "top/bottom".

- [Logging in](https://zulip.com/help/logging-in)
<img width="532" alt="image" src="https://user-images.githubusercontent.com/2343554/179670866-c1e71f23-641f-4ee7-8569-0dcbe94764e6.png">

- [Logging out](https://zulip.com/help/logging-out)
<img width="526" alt="image" src="https://user-images.githubusercontent.com/2343554/179671282-707c23c3-273f-428e-b638-c6e3ae67000e.png">

- [Switching between organizations](https://zulip.com/help/switching-between-organizations)
<img width="531" alt="image" src="https://user-images.githubusercontent.com/2343554/179671233-2acbfbd5-edbf-442c-9d07-1c03a4755f82.png">

alya commented:
Nice! Let's be consistent about the bolding on menu names (Zulip/Window/View menu). I think **Window** menu is the right approach -- does that sound right?

drrosa commented:
> Nice! Let's be consistent about the bolding on menu names (Zulip/Window/View menu). I think "**Window** menu" is the right approach -- does that sound right?

Yes, that sounds right.

I also changed "Enter your Zulip URL" to  "Enter the Zulip URL of the organization" to make it consistent with:
```
$ git grep "Zulip URL"
join-a-zulip-organization.md:1. Go to the Zulip URL of the organization.
join-a-zulip-organization.md:1. Go to the Zulip URL of the organization.
```

<img width="534" alt="image" src="https://user-images.githubusercontent.com/2343554/180673144-253a04ad-230c-4a96-b1ab-21074e39cb00.png">

alya commented:
Nice! Let's make it **Add Organization** to match the capitalization in the menu. Looks good otherwise!

drrosa commented:
> Nice! Let's make it **Add Organization** to match the capitalization in the menu. Looks good otherwise!

Oops! Done :)

<img width="501" alt="image" src="https://user-images.githubusercontent.com/2343554/181110776-e74158e6-e6c6-44bf-9964-47e54e33c0a3.png">

alya commented:
@timabbott ready for your review.

alya commented:
We ended up also cleaning up the Desktop app instructions. :)

timabbott commented:
Merged, thanks @drrosa and @alya!
